docs/user-guide/isis/sans2d.ipynb

```diff
@@ -167,6 +166,45 @@
     "workflow[Filename[EmptyBeamRun]] = isis.data.sans2d_tutorial_empty_beam_run()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "981f1f28",
+   "metadata": {},
+   "source": [
+    "The beam center is not set yet.\n",
+    "Unless we have a previously computed value, we can do so now:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "064f87a4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "workflow[BeamCenter] = sc.vector([0, 0, 0], unit='m')\n",
```

Comment:
Yes. It could in principle also be used to set a default that then gets refined... and I think I should add a test to ensure that we still get the absolute (relative to [0,0,0]) beam center back in a such as case.

Reply:
I think this is in fact not working: If we set anything but [0,0,0] the beam-center finder(s) return a beam center relative to the previously set center, which is not what we want, I think?

Reply:
Please see update, as discussed on the call.